UIKItTemplate/Modules/NewsLine/Model/DataStorage.swift

```diff
@@ -0,0 +1,185 @@
+        .init(userName: "barbara 001", imageName: "girl4"),
+        .init(userName: "drink forever", imageName: "girl2"),
+        .init(userName: "masha", imageName: "girl3"),
+        .init(userName: "constraint 90", imageName: "girl4"),
+        .init(userName: "liver 15", imageName: "girl2")
+    ]
+
+    // MARK: - Posts
+
+    /// Posts
+    private let posts: [Post] = [
+        .init(
+            avatarImageName: "man1",
+            nickName: "go to Brazil",
+            imageNames: ["nature1", "nature2", "nature3"],
+            title: "ПОездка в бразилию подарит вам незабываемые впечатления. Бронируйте только сейчас",
+            timeStamp: "13 минут вперед"
+        ),
+        .init(
+            avatarImageName: "man2",
+            nickName: "homik",
+            imageNames: ["nature2"],
+            title: "Разведение хомяков в домашних условиях. Плодятся лучше кроликов!",
+            timeStamp: "13 минут назад"
+        ),
+        .init(
+            avatarImageName: "girl2",
+            nickName: "Kazakhstan_forever",
+            imageNames: ["nature4"],
+            title: "ПОездка в Казахстан подарит вам незабываемые впечатления. Бронируйте только сейчас",
+            timeStamp: "13 дней вперед"
+        ),
+        .init(
+            avatarImageName: "man4",
+            nickName: "go to Brazil",
+            imageNames: ["nature1", "nature3"],
+            title: "ПОездка в бразилию подарит вам незабываемые впечатления. Бронируйте только сейчас",
+            timeStamp: "13 минут вперед"
+        ),
+        .init(
+            avatarImageName: "man2",
+            nickName: "go to Brazil",
+            imageNames: ["nature1", "nature2", "nature4", "nature3"],
+            title: "ПОездка в бразилию подарит вам незабываемые впечатления. Бронируйте только сейчас",
+            timeStamp: "13 минут вперед"
+        ),
+        .init(
+            avatarImageName: "man1",
+            nickName: "go to Brazil",
+            imageNames: ["nature1"],
+            title: "ПОездка в бразилию подарит вам незабываемые впечатления. Бронируйте только сейчас",
+            timeStamp: "13 минут вперед"
+        )
+    ]
+    /// First post in post sequence
+    var firstPost: Post {
+        posts[0]
+    }
+
+    /// All posts except first one
+    var postsExceptFirst: [Post] {
+        posts.dropLast()
+    }
+
+    // MARK: - Recomendations
+
+    /// Recomendations
+    let recomendations: [Recomendation] = [
+        .init(userName: "hromoy-405", imageName: "girl2"),
+        .init(userName: "hromoy-405", imageName: "man3"),
+        .init(userName: "hromoy-405", imageName: "nature4"),
+        .init(userName: "hromoy-405", imageName: "man1"),
+        .init(userName: "hromoy-405", imageName: "girl4"),
+        .init(userName: "hromoy-405", imageName: "nature1"),
+    ]
+
+    // MARK: - Notifications
+
+    /// Headers
+    let headers = ["Сегодня", "На этой неделе"]
```

Comment:
тут надо наверное, чтобы по названию было понятно к чему относятся хедеры, как вариант
```suggestion
    let noticeTableHeaders = ["Сегодня", "На этой неделе"]
```

Reply:
Я в headerTitles переименовал